client/src/app/site/pages/meetings/pages/motions/components/motion-export-dialog/components/motion-export-dialog/motion-export-dialog.component.ts

```diff
@@ -129,6 +129,16 @@ export class MotionExportDialogComponent extends BaseUiComponent implements OnIn
     @ViewChild(MOTION_PDF_OPTIONS.ContinuousText)
     public continuousTextButton!: MatButtonToggle;
 
+    private _isCSVExport = false;
+
+    protected set isCSVExport(format: ExportFileFormat) {
+        this._isCSVExport = format === ExportFileFormat.CSV;
+    }
```

Comment:
This is weird behavior. Just do the check on assignment. 

Return and assignment values of getter/setters should always be the same.